eslint.config.js

```diff
@@ -12,26 +10,7 @@ export default [
             "tools/create-test-example.js"
         ]
     },
-
-    // eslintConfigESLint[3] is eslint-plugin-n's recommended-script config
-    ...eslintConfigESLint.slice(0, 3),
-    {
-        files: ["**/*.js"],
-        ...nodeRecommendedModule
-    },
-    {
-        files: ["**/*.cjs"],
-        ...nodeRecommendedScript
-    },
-    ...eslintConfigESLint.slice(4),
-
-    {
-        settings: {
-            jsdoc: {
-                mode: "typescript"
-            }
-        }
-    },
```

Comment:
This is now included in eslint-config-eslint (https://github.com/eslint/eslint/pull/17338).